charts/linkerd-control-plane/templates/psp.yaml

```diff
@@ -37,6 +37,15 @@ spec:
     {{- else }}
     rule: RunAsAny
     {{- end }}
+  runAsGroup:
+    {{- if .Values.cniEnabled }}
+    rule: MustRunAs
+    ranges:
+    - min: 1000
+      max: 999999
+    {{- else }}
+    rule: RunAsAny
+    {{- end }}
```

Comment:
Would you mind explaining this in a sentence or two? Is this to allow the CNI plugin to use arbitrary GIDs?

Reply:
This is meant to prevent group IDs < 1000 if the cni is enabled (no privileged group IDs if cni is enabled) whereas any group ID is fine if cni is not used.